apps/web/package.json

```diff
@@ -164,8 +164,8 @@
     "storybook": "^7.4.2",
     "typescript": "4.9.5",
     "webpack": "5.78.0",
-    "webpack-dev-server": "4.11.1",
-    "webpack-bundle-analyzer": "^4.9.0"
+    "webpack-bundle-analyzer": "^4.9.0",
+    "webpack-dev-server": "4.11.1"
```

Comment:
please update the pnpm lock file one more time, locally it generates a new changes for me